lib/pointing_party_web/live/card_live.ex

```diff
@@ -0,0 +1,114 @@
+defmodule PointingPartyWeb.CardLive do
+  use Phoenix.LiveView
+
+  alias PointingParty.{Card, VoteCalculator}
+  alias PointingPartyWeb.{Endpoint, Presence}
+
+  def render(assigns) do
+    Phoenix.View.render(PointingPartyWeb.CardView, "index.html", assigns)
+  end
+
+  def mount(%{username: username}, socket) do
+    Endpoint.subscribe("users")
+    {:ok, _} = Presence.track(self(), "users", username, %{points: nil})
+    assigns = [
+      current_card: nil,
+      is_driving: false,
+      outcome: nil,
+      party_has_started: false,
+      remaining_cards: [],
+      results: nil,
+      username: username,
+      users: Presence.list("users"),
+      votes: []
+    ]
+
+    {:ok, assign(socket, assigns)}
+  end
+
+  def handle_event("next_card", points, socket) do
+    updated_socket = save_vote_next_card(points, socket)
+    Presence.update(self(), "users", updated_socket.assigns.username, &Map.put(&1, :points, nil))
+    Endpoint.broadcast_from(self(), "users", "update_card", %{card: updated_socket.assigns.current_card})
+
+    {:noreply, updated_socket}
+  end
+
+  def handle_event("start_party", _, socket) do
+    [current_card | remaining_cards] = Card.cards()
+    Endpoint.broadcast("users", "party_started", %{card: current_card})
+
+    {:noreply, assign(socket, is_driving: true, remaining_cards: remaining_cards)}
+  end
+
+  def handle_event("vote", %{"points" => points}, socket) do
+    Presence.update(self(), "users", socket.assigns.username, &Map.put(&1, :points, points))
+
+    if everyone_voted?() do
+      Endpoint.broadcast("users", "everyone_voted", %{})
```

Comment:
I think we can get rid of the `"everyone_voted"` broadcast, especially if we remove the concept of the "driver". When a user votes, LiveView should determine if everyone voted. If so, then calculate the winner/tie outcome and broadcast that "winner"/"tie" event.

Reply:
I kept the `"everyone_voted"` broadcast, but I consolidated some of the work. Take a look at the code and let me know what you think.